test-plone-5.1.x.cfg

```diff
@@ -0,0 +1,12 @@
+[buildout]
+extends =
+    https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-5.1.x.cfg
+    sources.cfg
+
+package-name = ftw.mobile
+
+auto-checkout =
+    plonetheme.blueberry
+
+[branches]
+plonetheme.blueberry = plone51
```

Comment:
🚧

Reply:
The branch `plone51` of `plonetheme.blueberry` has been merged and released as 1.11.0 on March 6, 2019.